core/src/main/web/app/utils/basic/commonui.js

```diff
@@ -201,11 +201,34 @@
         };
         $scope.hasCustomMarkup = function(item) {
           return typeof _.result(item, 'markup') !== 'undefined';
-        }
+        };
 
         $scope.getCustomMarkup = function(item) {
           return $sce.trustAsHtml(_.result(item, 'markup') || '');
-        }
+        };
+
+        $scope.getShortcut = function(itemShortcut) {
+         function replace(str) {
+            if (!bkHelper.isWindows) {
+              var mapObj = {
+                Cmd: "&#x2318;",
+                Alt: "&#x2325;"
+              };
+              str = str.replace(/Cmd|Alt/gi, function(matched){
```

Comment:
instead of duplicating the Cmd and Alt, should loop over the keys of the mapObj and apply substitution to each one.